docs/stories/components/bal-list/bal-list.stories.ts

```diff
@@ -78,7 +78,7 @@ export const NativeList = Story({
     <li>List Item 1</li>
     <li>List Item 2</li>
   </ul>
-  <ul class="list has-bullet-circle">
+  <ul class="list has-bullet-arrow-down">
```

Comment:
we should not replace the bullet circle style and more add the new style as a seperate list to show all variants